common/changes/@itwin/tree-widget-react/tree_widget_update_rulesets_2023-07-18-13-06.json

```
@@ -3,8 +3,8 @@
     {
       "packageName": "@itwin/tree-widget-react",
       "comment": "`ModelsTree`: Always render checkbox to avoid UI shifting when checkbox appear.",
-      "type": "none"
+      "type": "minor"
```

Isn't this a bugfix?

```suggestion
      "type": "patch"
```